pydoctor/themes/readthedocs/readthedocstheme.css

```diff
@@ -0,0 +1,730 @@
+#splitTables > table tr td:nth-child(2), .fieldArg {
+    font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+}
+
+code {
+    background: transparent;
+}
+
+/* Header */ 
+
+.page-header > h1 {
+    margin-top: 15px;
+    font-size: 28px;
+}
+
+.page-header > h1 code {
+    padding: 0;
+    font-size: 16px;
+}
+
+.page-header > h1 > code {
+    display: block;
+    line-height: 1.6;
+}
+
+.page-header > h1 code > a {
+    font-size: 16px;
+}
+
+/* Titles */
+
+p.caption, h1, h2, h3, h4, h5, h6, legend, .categoryHeader {
+
+    font-weight: bold;
+    font-family: "Roboto Slab",Georgia,Arial,sans-serif;
+}
+
+.categoryHeader{
+    font-size: 20px;
+    margin: 15px 0 20px 0;
+}
+
+/* Remove anchor hack */
+a[name] {
+    position: initial;
+} 
+
+
+/* Top-left navigation links */
+.navlinks {
+    background-color: rgb(41, 128, 185);
+    color: rgb(252, 252, 252)!important;
+    display: block;
+    padding: 20px 0 14px 0;
+    text-align: center;
+    width: 300px;
+}
+
+.navbar-brand {
+    display: block;
+}
+
+.navbar-brand >  a {
+    color: rgb(252, 252, 252)!important;
+    padding: 3px 5px!important;
+}
+
+.navbar-brand >  a.projecthome::before {
+    background-image: url(fonts/home.svg);
+}
+
+.navbar-brand >  a:last-child::before {
+    background-image: url(fonts/book.svg);
+}
+
+.navbar-brand >  a.projecthome::before, 
+.navbar-brand >  a:last-child::before {
+    background-size: 18px 15px;
+    background-position: center;
+    background-repeat: no-repeat;
+    display: inline-block;
+    width: 20px; 
+    height: 15px;
+    content:"";
+    -webkit-filter: invert(100%);
+    filter: invert(100%);
+    margin-bottom: -2px;
+}
+
+.navbar-brand > a:hover, .navlinks > a:hover {
+    background-color:rgba(255, 255, 255, 0.1);
+    text-decoration: none;
+}
+
+.navlinks > a {
+    color: rgba(210, 239, 255, 1);
+    display: inline-block;
+    height: 32px;
+    line-height: 32px;
+    white-space: nowrap;
+    padding: 5px!important;
+    margin-left: 5px;
+    margin-top: 5px
+}
+
+
+/* Generics for links */
+a, a:visited {
+    text-decoration:none;
+}
+a:hover{
+    text-decoration: underline;
+}
+
+/* Part of bootstrap CSS: buttons */
+.btn {
+    cursor: pointer;
+}
+
+.btn-link, a {
+    font-weight: 400;
+    color: #337ab7;
+    border-radius:0;
+}
+
+.btn-link, .btn-link.active, .btn-link:active {
+    background-color: transparent;
+    -webkit-box-shadow: none;
+    box-shadow:none
+}
+
+.btn-link, .btn-link:active, .btn-link:focus, .btn-link:hover {
+    border-color:transparent
+}
+
+.btn-link:focus, .btn-link:hover {
+    color: #23527c;
+    background-color:transparent
+}
+
+/* Summary tables */
+
+#splitTables > table {
+    -webkit-border-horizontal-spacing: 0px;
+    -webkit-border-vertical-spacing: 0px;
+    border: 1px solid rgb(210, 220, 216);
+    border-bottom: 0;
+    vertical-align: middle;
+}
+
+#splitTables > table td {
+    border-bottom: 1px solid rgb(210, 220, 216);
```

Comment:
I don't really like that border in combination with the `border-spacing` of the table. It looks like each cell has a small shadow.

![image](https://user-images.githubusercontent.com/73739153/153351551-fcbcbf2e-92aa-4418-882b-0b90d34d7882.png)

Reply:
Fixed, Thanks